### ISSUE TITLE
evaluate运行时出现问题

### ISSUE THREAD

**Issue by yfwang214** [yeyupiaoling/Whisper-Finetune#56]
A custom logits processor of type <class 'transformers.generation.logits_process.ForceTokensLogitsProcessor'> with values <transformers.generation.logits proces.ForceTokensLogitsProcessor object at x7fcf47228c10> has been passed to ’.generate()’, but it has already been created with the values <transfomers.generation.logits process .ForceTokenslogitsProcessor object at 0xfcf4728df0>. <transformers.generation.logits pracess.forceTokenslogitsProcessor object at 8x7fcf47228df0> has ben created by passing the corresponding arguments to generate or by the model's config default valves. If you just want to change the defaut values of logits procesor consider passing them as arguments to '.generate()'instead of using a custom logits processor.

请问该怎么解决？

**Comment by yeyupiaoling** [comment]
@yfwang214 你是改了什么？或者是数据输入 格式有问题，还是下载的模型不对？

**Comment by yfwang214** [comment]
已经解决了，谢谢

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年02月24日 11:19 |
| 收件人 | yeyupiaoling/Whisper-Finetune ***@***.***> |
| 抄送人 | yfwang214 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [yeyupiaoling/Whisper-Finetune] evaluate运行时出现问题 (Issue #56) |

@yfwang214 你是改了什么？或者是数据输入 格式有问题，还是下载的模型不对？

**Comment by yeyupiaoling** [comment]
@yfwang214 好的

**Comment by yfwang214** [comment]
您好，之前跟您学习了whisper的微调，如果我想要不微调这个模型，训练我自己的数据，与微调的cer做比较，请问我是不是要修改finetune的代码进行训练？还是说要如何进行？谢谢

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年02月24日 11:22 |
| 收件人 | yeyupiaoling/Whisper-Finetune ***@***.***> |
| 抄送人 | yfwang214 ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [yeyupiaoling/Whisper-Finetune] evaluate运行时出现问题 (Issue #56) |

@yfwang214 好的

**Comment by Koyonyong** [comment]
> A custom logits processor of type <class 'transformers.generation.logits_process.ForceTokensLogitsProcessor'> with values <transformers.generation.logits proces.ForceTokensLogitsProcessor object at x7fcf47228c10> has been passed to ’.generate()’, but it has already been created with the values <transfomers.generation.logits process .ForceTokenslogitsProcessor object at 0xfcf4728df0>. <transformers.generation.logits pracess.forceTokenslogitsProcessor object at 8x7fcf47228df0> has ben created by passing the corresponding arguments to generate or by the model's config default valves. If you just want to change the defaut values of logits procesor consider passing them as arguments to '.generate()'instead of using a custom logits processor.
> 
> 请问该怎么解决？

您好，我也遇到了这个问题，我没有改动这个代码的任何部分，测试的model是"openai/whisper-large-v2"，请问是怎么解决的

**Comment by TYLS3125** [comment]
> A custom logits processor of type <class 'transformers.generation.logits_process.ForceTokensLogitsProcessor'> with values <transformers.generation.logits proces.ForceTokensLogitsProcessor object at x7fcf47228c10> has been passed to ’.generate()’, but it has already been created with the values <transfomers.generation.logits process .ForceTokenslogitsProcessor object at 0xfcf4728df0>. <transformers.generation.logits pracess.forceTokenslogitsProcessor object at 8x7fcf47228df0> has ben created by passing the corresponding arguments to generate or by the model's config default valves. If you just want to change the defaut values of logits procesor consider passing them as arguments to '.generate()'instead of using a custom logits processor.
> 
> 请问该怎么解决？
> 
> 已将类型为类“transformers.generation.logits_process.ForceTokensLogitsProcessor”且值为transformers.generation.logits process.ForceTokensLogitsProcessor对象的自定义logits处理器传递给<transformers.generation.logits pracess. forceTokensystemitsProcessor对象在8x7fcf47228df 0已经通过传递相应的参数来生成或通过模型的配置默认值创建。如果你只是想改变logits处理器的默认值，或者考虑将它们作为参数传递给“.generate（n）”，而不是使用自定义logits处理器。
> 
> 请问该怎么解决？
怎么解决啊，大佬

**Comment by TYLS3125** [comment]
> 已经解决了，谢谢
> […](#)
> ---- 回复的原邮件 ---- | 发件人 | ***@***.***> | | 发送日期 | 2024年02月24日 11:19 | | 收件人 | yeyupiaoling/Whisper-Finetune ***@***.***> | | 抄送人 | yfwang214 ***@***.***>, Mention ***@***.***> | | 主题 | Re: [yeyupiaoling/Whisper-Finetune] evaluate运行时出现问题 (Issue #56) | @yfwang214 你是改了什么？或者是数据输入 格式有问题，还是下载的模型不对？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***>

怎么解决的，大佬？
